tests/Framework.py

```diff
@@ -240,17 +245,9 @@ def __request_callback(self, request, uri, response_headers):
         output = bytearray(readLine(self.__file), "utf-8")
         readLine(self.__file)
 
-        # make a copy of the headers and remove the ones that interfere with the response handling
-        adding_headers = CaseInsensitiveDict(self.response_headers)
-        adding_headers.pop("content-length", None)
-        adding_headers.pop("transfer-encoding", None)
-        adding_headers.pop("content-encoding", None)
-
-        response_headers.update(adding_headers)
         return [status, response_headers, output]
```

Comment:
Can you elaborate on why this is not needed anymore?

Reply:
It has been a while since I wrote this code and I am not sure if I remember it correctly even because I have never really used `responses` but I found it quite nice to work with. I am just trying to get rid of managing `httpretty` in Fedora.
 
I think the reason is that I was finding more headers that somehow were *interfering with the response handling*, I have solved it in the other way around; checking just a subset of the headers [here](https://github.com/PyGithub/PyGithub/blob/429569c6f5f6c1bbfeadc129e0958506c892ac27/tests/Framework.py#L235-L238).